tasks/cluster_agent.py

```diff
@@ -92,7 +92,7 @@ def integration_tests(ctx, race=False, remote_docker=False, go_mod="readonly", t
     Run integration tests for cluster-agent
     """
     if sys.platform == 'win32':
-        raise Exit(message='cluster-agent integration tests are not supported on Windows', code=0)
+        raise NotImplementedError('Cluster Agent integration tests are not supported on Windows')
```

Comment:
I would recommend using a dedicated exception, since we're attaching a custom behavior to it (if anything else ends up throwing this, we'd end up with untraceable error in the logs)

Reply:
Good point, fixed with [06c1ef0](https://github.com/DataDog/datadog-agent/pull/31740/commits/06c1ef052f7f827874b0d614b06ceb9286042303).